docs/getting-started/getting-started-with-low-code-sql.md

```diff
@@ -389,6 +389,6 @@ You've successfully set up, developed, tested, and deployed your first SQL proje
 
 To continue learning and expanding your skills with Prophecy, explore other tutorials within our documentation, or apply your newfound knowledge to address real-world business challenges!
 
-If you ever encounter any difficulties, don't hesitate to reach out to us at Support@Prophecy.io or join our [Slack community](https://prophecy-io-support.slack.com/join/shared_invite/zt-moq3xzoj-~5MSJ6WPnZfz7bwsqWi8tQ#/shared-invite/email) for assistance. We're here to help!
+If you ever encounter any difficulties, don't hesitate to Contact.us@Prophecy.io or join our [Slack community](https://prophecy-io-support.slack.com/join/shared_invite/zt-moq3xzoj-~5MSJ6WPnZfz7bwsqWi8tQ#/shared-invite/email) for assistance. We're here to help!
```

Comment:
I still think you need "don't hesitate to reach out to us at" in front of the email, unless we're being clever using the email address, in which case disregard